playbooks/tasks/portal-role-task-mongo-check-docker-service.yml

```diff
@@ -80,7 +76,7 @@
   vars:
     mongodb_shell:
       eval: "db.serverStatus()"
-      until: "'\"ok\" : 1' in mongo_shell_result.transformed_output"
+      until: '''"ok" : 1'' in mongo_shell_result.transformed_output'
```

Comment:
same

Reply:
this was automatically transformed by prettier in my vscode, I tested this playbook so if there would be any issue it would not pass successfully I guess

Reply:
same